apps/nextjs-website/react-components/components/Footer/helpers/FooterColumn.tsx

```diff
@@ -32,7 +32,7 @@ export const FooterColumn = ({
       >
         {icons.map(({ iconURL, href = hrefNoOp, ariaLabel }, i) => (
           <li key={i}>
-            <Link href={href} aria-label={ariaLabel}>
+            <Link href={href} aria-label={ariaLabel} target='_blank'>
```

Comment:
```suggestion
            <Link href={href} aria-label={ariaLabel} {...(href.startsWith('https') && { target: '_blank' })}>
```
Only apply target=_blank to external links